timelock-server/src/testCommon/java/com/palantir/atlasdb/timelock/TestableTimelockCluster.java

```diff
@@ -56,35 +52,28 @@
     @Nullable
     private TestableTimelockServer lastLeader;
 
-    public TestableTimelockCluster(String baseUri, String... configFileTemplates) {
-        this(ClusterName.random(), baseUri, configFileTemplates);
+    public TestableTimelockCluster(String configFileTemplate, TemplateVariables... variables) {
+        this(name(), configFileTemplate, variables);
+    }
+
+    public TestableTimelockCluster(String name, String configFileTemplate, TemplateVariables... variables) {
+        this(name, configFileTemplate, ImmutableList.copyOf(variables));
     }
 
-    public TestableTimelockCluster(ClusterName clusterName, String baseUri, String... configFileTemplates) {
-        this.clusterName = clusterName.get();
-        this.configs = Arrays.stream(configFileTemplates)
-                .map(this::getConfigHolder)
+    public TestableTimelockCluster(String name, String configFileTemplate, Iterable<TemplateVariables> variables) {
+        this.name = name;
+        this.configs = Streams.stream(variables)
```

Comment:
Can we change this to `Set<TemporaryConfigurationHolder>`?

Reply:
any particular reason why?

Reply:
@OStevan ping on this